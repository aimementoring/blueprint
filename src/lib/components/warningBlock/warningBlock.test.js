import React from 'react';
import renderer from 'react-test-renderer';
import WarningBlock from './warningBlock.js';

describe('WarningBlock', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <WarningBlock>
          Ups! There was an error, please call your AIME key contact for more information
        </WarningBlock>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
