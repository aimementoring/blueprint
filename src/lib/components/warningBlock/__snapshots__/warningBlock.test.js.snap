// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WarningBlock renders properly 1`] = `
<div
  className="theme-base"
>
  <div
    className="warningBlock "
  >
    Ups! There was an error, please call your AIME key contact for more information
  </div>
</div>
`;
